tools/topology/topology2/include/dais/pdm_config.conf

```diff
@@ -0,0 +1,55 @@
+Class.Base."pdm_config" {
+	#
+	# Argument used to construct DMIC PDM config
+	#
+	DefineAttribute."ctrl_id" {
+		token_ref	"sof_tkn_dmic_pdm.short"
+	}
+
+	DefineAttribute.mic_a_enable {
+		token_ref	"sof_tkn_dmic_pdm.short"
+	}
+
+	DefineAttribute.mic_b_enable {
+		token_ref	"sof_tkn_dmic_pdm.short"
+	}
+
+	DefineAttribute.polarity_a {
+		token_ref	"sof_tkn_dmic_pdm.short"
+	}
+
+	DefineAttribute.polarity_b {
+		token_ref	"sof_tkn_dmic_pdm.short"
+	}
+
+	DefineAttribute.clk_edge {
+		token_ref	"sof_tkn_dmic_pdm.short"
+	}
+
+	DefineAttribute.skew {
+		token_ref	"sof_tkn_dmic_pdm.short"
+	}
+
+	attributes {
+		constructor [
```

Comment:
please add `!` before `constructor` and `mandatory`, and make them `!constructor` and `!mandatory`, for the reason, please check https://github.com/alsa-project/alsa-lib/issues/151

Reply:
ok added now